iocore/net/QUICNetVConnection.cc

```diff
@@ -1577,14 +1570,18 @@ QUICNetVConnection::_packetize_frames(QUICEncryptionLevel level, uint64_t max_pa
           }
         }
 
-        if (!ack_eliciting && frame->type() != QUICFrameType::ACK) {
+        if (!ack_eliciting && frame->type() != QUICFrameType::PADDING && frame->type() != QUICFrameType::ACK) {
```

Comment:
Maybe it's time to introduce `bool QUICFrame::is_ack_eliciting()`?

Reply:
done